earth_enterprise/tutorial/SConstruct

```diff
@@ -36,8 +36,7 @@ root_dir = tutorials_dir + '/opt/google/share/tutorials/fusion/'
 
 # Get the current directory, since SCons will move into the directory
 #   it's using, and FusionTutorial is expected below the SConscript file.
-tutorial_src_dir = os.sep.join((os.getcwd().split(os.sep) +
-                    ['FusionTutorial']))
+tutorial_src_dir = os.path.join(os.getcwd(), 'FusionTutorial'])
```

Comment:
This seems to be a syntax error now...  Need to fix this.  The staging is broken.  Unfortunately, the Travis builder doesn't run the stage command.

Reply:
Roger. I found it here before I saw your comment. It's fixed as of fc6d9a8.